### ISSUE TITLE
[salt-cloud lxc] --destroy stop after destroying first machine

### ISSUE THREAD

**Issue by arthurlogilab** [saltstack/salt#24037]
```
# salt-cloud -m map_lxc_test --destroy
[INFO    ] Runner completed: 20150521181202068624
The following virtual machines are set to be destroyed:
  ursa-lxc:
    lxc:
      one
      two
      three
Proceed? [N/y] y
... proceeding
[INFO    ] Destroying in non-parallel mode.
ursa:
    True
[INFO    ] Runner completed: 20150521181210556479
ursa:
[snip]
    size:
        239932952
    state:
        running
[INFO    ] Runner completed: 20150521181211790308
ursa:
    ----------
    result:
        True
    state:
        ----------
        new:
            None
        old:
            running
[INFO    ] Runner completed: 20150521181214852912
[ERROR   ] There was an error destroying machines: 'change'
# lxc-ls --active
two three
# salt-run --versions-report
                  Salt: 2015.5.0
                Python: 2.7.6 (default, Mar 22 2014, 22:59:56)
                Jinja2: 2.7.2
              M2Crypto: 0.21.1
        msgpack-python: 0.3.0
          msgpack-pure: Not Installed
              pycrypto: 2.6.1
               libnacl: Not Installed
                PyYAML: 3.10
                 ioflo: Not Installed
                 PyZMQ: 14.0.1
                  RAET: Not Installed
                   ZMQ: 4.0.4
                  Mako: 0.9.1
 Debian source package: 2015.5.0+ds-1trusty1

```

**Comment by arthurlogilab** [comment]
Running in parallel mode seems to work and shows a traceback that might be a clue to why it stops on the first container : 

```
TRACE: 'change'
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/salt/cloud/__init__.py", line 55, in _call
    ret = func(*args, **kw)
  File "/usr/lib/python2.7/dist-packages/salt/cloud/__init__.py", line 2205, in destroy_multiprocessing
    output = fun(parallel_data['name'])
  File "/usr/lib/python2.7/dist-packages/salt/cloud/clouds/lxc.py", line 380, in destroy
    ret['result'] = cret['change']
KeyError: 'change'
```

**Comment by rallytime** [comment]
This was fixed by #23770. Closing.
